backend-comparison/src/burnbenchapp/mod.rs

```diff
@@ -12,3 +12,12 @@ use tui::TuiApplication as App;
 mod term;
 #[cfg(not(feature = "tui"))]
 use term::TermApplication as App;
+
+const BENCHMARKS_TARGET_DIR: &str = "target/benchmarks";
+const USER_BENCHMARK_SERVER_URL: &str = if cfg!(debug_assertions) {
+    // development
+    "http://localhost:8000/"
+} else {
+    // production
+    "https://user-benchmark-server-gvtbw64teq-nn.a.run.app/"
+};
```

Comment:
We should add somewhere that running burnbench app without the --release flag won't update the results to the server.

Reply:
I updated the README and all the snippets to include the --release flag